### ISSUE TITLE
Editing cards in Emacs

### ISSUE THREAD

**Issue by nanjigen** [chenyanming/anki.el#4]
Is it possible to edit the `shrface` rendered card in its Emacs buffer and have the results write back to the Anki database?

**Comment by chenyanming** [comment]
Simple editing is possible (Update existing cards' content), but complicated editing is probably not in `anki.el`. I am planning to call Anki Official App to do the editing job in the next version.

**Comment by chenyanming** [comment]
Finally, I plan to work on Adding/Updating/Deleting on Anki Database directly, this needs some work and a lot of tests.

**Comment by doolio** [comment]
You are probably aware of [anki-editor](https://github.com/louietan/anki-editor). Perhaps it may be of use￼ or could be combined with this project? 

Is there an argument to not rely on the Anki application database (or on Anki at all) but instead create your own. I really like Anki but I always have this (perhaps unfounded) fear that my large database gets corrupted and I lose access to my cards and the work I put into creating them. That's why solutions within Emacs appeal to me and to be able to create the cards as plaintext as I don't have to fear them getting corrupted. 

The one drawback I see with plaintext cards is that the file would get large pretty quickly if one adheres to the recommendation to keep everything in a single deck (one file) rather than splitting the information into multiple decks (separate files).

**Comment by chenyanming** [comment]
Yes. It is possible, now anki.el have ability to read the Anki database but not write. I can provide an import function for it. But I am afraid that it can not import back to Anki database.

But about the card edit feature, it still needs to be implemented. Maybe combine with anki editor.
